scraper/src/github-scraper/config.ts

```diff
@@ -0,0 +1,11 @@
+import { Octokit } from "octokit";
+
+export const GITHUB_TOKEN = process.env.GITHUB_TOKEN;
```

Comment:
`GITHUB_TOKEN` is required only by octokit and octokit is already exported. Why is the `GITHUB_TOKEN` exported too?

```suggestion
const GITHUB_TOKEN = process.env.GITHUB_TOKEN;
```